src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Publish.targets

```diff
@@ -1,5 +1,6 @@
 <Project ToolsVersion="12.0" DefaultTargets="PublishOutputLeg" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <UsingTask TaskName="PushToBlobFeed" AssemblyFile="$(MSBuildThisFileDirectory)Microsoft.DotNet.Build.Tasks.Feed.dll"/>
+  <Import Project="$(MSBuildThisFileDirectory)Common.props" />
+  <UsingTask TaskName="PushToBlobFeed" AssemblyFile="$(_TaskDir)Microsoft.DotNet.Build.Tasks.Feed.dll"/>
```

Comment:
Why not just do the UsingTask instead Common?

Reply:
Got rid of Common.props